meshmode/discretization/poly_element.py

```diff
@@ -518,11 +518,16 @@ def from_mesh_interp_matrix(self):
 
 class TensorProductElementGroupBase(PolynomialElementGroupBase,
         HypercubeElementGroupBase):
-    def __init__(self, mesh_el_group, order, index=None, *, basis, unit_nodes):
+    def __init__(self, mesh_el_group, order, index=None, *, basis, basis_1d,
+                 unit_nodes, unit_nodes_1d):
         """
         :arg basis: a :class:`modepy.TensorProductBasis`.
+        :arg basis_1d: a representation of the 1D basis from which the tensor
+        product basis is built.
         :arg unit_nodes: unit nodes for the tensor product, obtained by
             using :func:`modepy.tensor_product_nodes`, for example.
+        :arg unit_nodes_1d: the one-dimensional set of nodes used in the tensor
+        product, i.e. the set of nodes from which `unit_nodes` is constructed.
```

Comment:
```suggestion
        :arg basis_1d: a representation of the 1D basis from which the tensor
        	product basis is built.
        :arg unit_nodes: unit nodes for the tensor product, obtained by
            using :func:`modepy.tensor_product_nodes`, for example.
        :arg unit_nodes_1d: the one-dimensional set of nodes used in the tensor
        	product, i.e. the set of nodes from which `unit_nodes` is constructed.
```
These need to be indented for Sphinx to parse it properly (I think?)